backend/src/panels/AbstractWebviewPanel.ts

```diff
@@ -35,7 +35,7 @@ export abstract class AbstractWebviewPanel {
 		this.extensionPath = context.extensionPath;
 		this.mediaPath = path.join(context.extensionPath, "dist", "media");
 		this.htmlFileName = "index.html";
-		this.logger = getLogger();
+		this.logger = getClassLogger("AbstractWebviewPanel");
```

Comment:
```suggestion
		this.logger = getClassLogger(this.constructor.name);
```

Reply:
It doesn't work after minimizing.
I always did it in this way (this.constructor.name) and now the projects which were minimized don't show class names and functions names properly.
So we should do it hard-coded.